cli/cli_main.cpp

```diff
@@ -36,11 +36,13 @@
 #include <iomanip>
 #include <iostream>
 
+#include "../version.h"
```

Comment:
Relative paths don't work for out-of-source builds.  You'll need to ensure CMAKE_BINARY_DIR is added to include path.  That's why many builds failed.